### DISCUSSION TITLE
debug/debugger 的翻譯

### DISCUSSION THREAD

**Discussion by mattwang44** [python/python-docs-zh-tw#513] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

debug/debugger

### 術語翻譯

「除錯/除錯器」或「偵錯/偵錯器」

### 說明

- 出處：https://docs.python.org/3/library/pdb.html
- 翻譯：「除錯/除錯器」或「偵錯/偵錯器」我個人沒有特別偏好，但想討論是否需要統一用詞；曾聽過一種說法是 debugger 是只能夠「偵錯」，但人類去 debug 就是真的「除錯」，故也需要根據上下文判斷。

**Comment by xchux** [comment]
依照現有項目 debugger 翻譯，也有用「偵錯工具」，也提供參考
ref.
- [microsoft visualstudio](https://learn.microsoft.com/zh-tw/visualstudio/debugger/what-is-debugging?view=vs-2022)
- [android studio](https://developer.android.com/studio/debug?hl=zh-tw)
